utils/connection.py

```
@@ -26,4 +32,32 @@ def connectDB(sql):
     df = conn.query(sql, ttl=0)
-    return df.to_dict(orient="records")
\ No newline at end of file
+    
+    return df.to_dict(orient="records")
+
+
+def updateDB(sqlList, paramList):
+    print("update Start!")
+    if os.getenv("GITHUB_ACTIONS") != "true":
+        load_dotenv()
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)